package.json

```diff
@@ -91,7 +91,7 @@
     "@bbc/psammead-inline-link": "^0.3.4",
     "@bbc/psammead-media-indicator": "^0.2.1",
     "@bbc/psammead-paragraph": "^1.0.2",
-    "@bbc/psammead-section-label": "^0.1.0-alpha.0",
+    "@bbc/psammead-section-label": "^0.1.1",
     "@bbc/psammead-sitewide-links": "^0.3.9",
     "@bbc/psammead-story-promo": "^0.2.0-alpha.6",
     "@bbc/psammead-story-promo-list": "^0.1.0-alpha.2",
```

Comment:
Can you bump story promo list here please?

Reply:
Updated it.